go.mod

```diff
@@ -22,10 +22,12 @@ require (
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/opencontainers/runc v0.1.1 // indirect
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
+	github.com/pkg/errors v0.8.1
```

Comment:
I searched in the project and seems that there isn't any `github.com/pkg/errors`. I think this can be removed.

Reply:
Fixed !